frame/support/test/tests/benchmark_ui/pass/valid_result.rs

```diff
@@ -0,0 +1,18 @@
+use frame_benchmarking::v2::*;
+use frame_support_test::Config;
+
+#[benchmarks]
+mod benches {
+	use super::*;
+
+	#[benchmark]
+	fn bench() -> Result<bool, BenchmarkError> {
```

Comment:
This might be a question for @ggwpez as well, but is there any way that returning a non-`()` `Ok` value makes sense in a benchmark function? I struggle to see how it could be used...

Reply:
Yeah what I imagined is it might be useful for quickly manually testing something but I do agree, I merely put this here in case someone needs it because otherwise there is no way to return a value.

Reply:
We don't use the returned value in the benchmarking, so from a FRAME side: no.  
Besides than that i dont see it being harmful to have :man_shrugging:

Reply:
Yeah it's probably not worth it. I'll just drop support for this. Alernatively I could constraint `T` but it doesn't look like anyone will need this

Reply:
Ok it has been removed